testing/regress/ecl/regexfindset.ecl

```diff
@@ -0,0 +1,51 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2016 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+
+/* ascii strings */
+str := 'add-subtract 123.90, or 123';
+REGEXFINDSET(NOFOLD('(([[:alpha:]]|(-))+)|([[:digit:]]+([.][[:digit:]]+)?)'), str);
+
+REGEXFINDSET('(([[:alpha:]]|(-))+)|([[:digit:]]+([.][[:digit:]]+)?)', str);
+
+/* Unicode strings */
+ustr := U'add-subtract 123.90, or 123';
+REGEXFINDSET(NOFOLD(U'(([[:alpha:]]|(-))+)|([[:digit:]]+([.][[:digit:]]+)?)'), ustr);
+
+REGEXFINDSET(U'(([[:alpha:]]|(-))+)|([[:digit:]]+([.][[:digit:]]+)?)', ustr);
+
+/* in Project test */
+RecLayout := RECORD
+    STRING50 rawstr;
+END;
+ParsedLayout := RECORD
+    SET OF STRING parsed;
+END;
+
+recTable := dataset([
```

Comment:
Note - this entire test is currently constant folded!
You need to place a NOFOLD() around the definition of recTable and urecTable below.

E.g.,

    recTable := NOFOLD(dataset([....], recLayout));